scripts/jenkins/ardana/manual/input.yml.example

```diff
@@ -155,6 +155,9 @@ swift_nodes: ''
 # disabled_services: 'monasca|logging|ceilometer|cassandra|kafka|spark|storm|freezer|octavia'
 disabled_services: ''
 
+# Enable (true) or disable (false) firewall rules on control plane. Default is enable: true
+firewall_enabled: 'true'
+
```

Comment:
Do you need to expose this parameter to Jenkins for user?  Same for scenario name - dualstack.

Reply:
Maybe at some point. This is first for manual deploy. I do think we'll have to expose it at some point. 
Yes dualstack works, however only CLM is placed this. This really adds a new ipv4 network to put rabbitmq and mysql, but since that would have created more networks than necessary, I just thought to use the same network for CLM.

Reply:
But I think if we can get mariadb working on ipv6 before this is done.  so perhaps we can call it something like ipv6 with rabbitmq ipv4 model. 
The point of this patch is just to have something that can be used for developers currently to develop on ipv6.  If we find another service that can only run on ipv4 we can then move it to the ipv4 side.